### DISCUSSION TITLE
Check for update by JS-SDK

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#19997] [Feature Requests]
### Summary

It would be great to check for updates and get more information about the current version of directus by JS SDK.

### Motivation

We want to create a status widget for tracking the instances. It would be important whether there is a update available.

### Detailed Design

Function in JS SDK as util to request the version and the update check.

### Must Have:
- current directus version
- update available as boolean

### Drawbacks

No reasons

### Alternatives

Collect informations manually

### Adoption Strategy

No breaking changes

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
